node-it/src/test/scala/com/wavesplatform/it/sync/smartcontract/DataTransactionBodyBytesByteVectorSuite.scala

```diff
@@ -0,0 +1,101 @@
+package com.wavesplatform.it.sync.smartcontract
+
+import com.typesafe.config.Config
+import com.wavesplatform.common.state.ByteStr
+import com.wavesplatform.common.utils.EitherExt2
+import com.wavesplatform.it.NodeConfigs
+import com.wavesplatform.it.api.SyncHttpApi._
+import com.wavesplatform.it.api.TransactionInfo
+import com.wavesplatform.it.sync.{setScriptFee, _}
+import com.wavesplatform.it.transactions.BaseTransactionSuite
+import com.wavesplatform.lang.v1.compiler.Terms
+import com.wavesplatform.lang.v1.estimator.v2.ScriptEstimatorV2
+import com.wavesplatform.state.{BinaryDataEntry, DataEntry}
+import com.wavesplatform.transaction.TxVersion
+import com.wavesplatform.transaction.smart.script.ScriptCompiler
+
+class DataTransactionBodyBytesByteVectorSuite extends BaseTransactionSuite {
+  private def compile(scriptText: String) =
+    ScriptCompiler.compile(scriptText, ScriptEstimatorV2).explicitGet()._1.bytes().base64
+
+  override protected def nodeConfigs: Seq[Config] =
+    NodeConfigs.newBuilder
+      .overrideBase(_.quorum(0))
+      .withDefault(1)
+      .buildNonConflicting()
+
+  private val maxDataTxV1bodyBytesSize = 153530
+  // actually lower than Terms.DataTxMaxBytes
```

Comment:
why?

Reply:
Because DataTransaction.MaxBytes is used to compare with whole tx bytes